server/pages/index.jade

```diff
@@ -44,6 +46,9 @@ html(lang=lang, dir=isRTL ? 'rtl' : 'ltr', class=isFluidWidth ? 'is-fluid-width'
 		link(rel='apple-touch-icon', sizes='180x180', href='//s1.wp.com/i/favicons/apple-touch-icon-180x180.png')
 		link(rel='profile', href='http://gmpg.org/xfn/11')
 		link(rel='manifest', href='/calypso/manifest.json')
+		
+		//- Preconnect to Google's font CDN. Note the mandatory crossorigin attr for fonts.
```

Comment:
I'm wondering if all those `preconnect` links shouldn't go to the top of the head. I don't know what are the rules here.

Reply:
Given that resource hints are meant to be consumed by the browser to help prioritize order of asset loading, I think having them in the `head` is appropriate.

Reply:
They definitely need to stay in the `head` :)

Reply:
Ah! I see. I'm not 100% sure on that part either - I defaulted to putting the resource hint right before the actual resource just in case.